src/components/staff-payment/StaffPayment.vue

```diff
@@ -287,6 +267,24 @@ export default class StaffPayment extends Vue {
     }
   }
 
+  /** Watches for change to FAS form validity. */
+  @Watch('fasFormValid')
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Instead of this, could you just have no formal parameters on line 273 (and 282)?

Reply:
I removed the parameters!